app/controllers/provider/admin/backend_apis/stats/base_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Provider::Admin::BackendApis::Stats::BaseController < Stats::BaseController
+  include ApiAuthentication::ByAccessToken
+  self.access_token_scopes = :stats
+
+  before_action :authorize_monitoring
+  before_action :find_backend_api
+
+  activate_menu :backend_api, :monitoring
+
+  sublayout :stats
+
+  def index
+    @metrics = @backend_api.metrics.top_level
+    @methods = @backend_api.method_metrics
+
+    respond_to do |format|
+      format.html { render :index }
+      format.json { render json: metrics_with_methods }
+    end
+  end
+
+  protected
+
+  # TODO: ensure provider domain
+  def authorize_monitoring
+    authorize! :manage, :monitoring
+  end
+
+  def find_backend_api
+    @backend_api = collection.find(params[:backend_api_id])
+    authorize! :show, @backend_api
```

Comment:
Wait, if both master and tenant can manage backend apis:
![image](https://user-images.githubusercontent.com/11318903/78252885-d1906e80-74f3-11ea-9d8f-6c31ceafbe76.png)

Then I do not understand the TODO comment that I spoke about in https://github.com/3scale/porta/pull/1793/files#r402297728

Reply:
In the end, I think everything related to `BackendApi` is going to be removed for master, including this ability.